### ISSUE TITLE
No sound is heard when "Jump to Min/Max" keyboard shortcuts are used

### ISSUE THREAD

**Issue by Nancy-Salpepi** [phetsims/fourier-making-waves#192]
**Test device**
MacBook Air (m1 chip)

**Operating System**
11.6

**Browser**
chrome

**Problem description**
https://github.com/phetsims/qa/issues/711

No sound is produced when the home/end keys are used to jump to min/max values. This is seen on the Discrete Screen for the number of Harmonics as well as Wavelength and Period values. This also occurs in the Wave Game Screen for the amplitude values.

**Steps to reproduce**
1. On the discrete screen, tab to the Harmonics spinner. 
2. Use the "Home" and "End" keys to adjust value to min and max. 

**Visuals**

https://user-images.githubusercontent.com/87318828/135151925-8cebf7ee-4227-4a21-842b-848551aacc32.mov

<details>
Troubleshooting information:
!!!!! DO NOT EDIT !!!!!
Name: ‪Fourier: Making Waves‬
URL: https://phet-dev.colorado.edu/html/fourier-making-waves/1.0.0-rc.1/phet/fourier-making-waves_all_phet.html
Version: 1.0.0-rc.1 2021-09-28 15:44:23 UTC
Features missing: applicationcache, applicationcache, touch
Flags: pixelRatioScaling
User Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/94.0.4606.61 Safari/537.36
Language: en-US
Window: 1391x690
Pixel Ratio: 2/1
WebGL: WebGL 1.0 (OpenGL ES 2.0 Chromium)
GLSL: WebGL GLSL ES 1.0 (OpenGL ES GLSL ES 1.0 Chromium)
Vendor: WebKit (WebKit WebGL)
Vertex: attribs: 16 varying: 31 uniform: 1024
Texture: size: 16384 imageUnits: 16 (vertex: 16, combined: 80)
Max viewport: 16384x16384
OES_texture_float: true
Dependencies JSON: {}
</details>

**Comment by pixelzoom** [comment]
The amplitude sliders use an adhoc implemenation for sound, intended as a temporary solution until common-code work is done for https://github.com/phetsims/sun/issues/697 (Slider sound API). This adhoc implementation is by no means complete.  It only supports sound when dragging (with a pointer or keyboard).  Clicking in the track (without dragging) or moving to the min/max with the keyboard is not at all supported.  

@arouinfar let me know if you want anything else added to this adhoc implementation. If not, please close.

**Comment by pixelzoom** [comment]
9/30/21 design meeting:

We're not going to address this issue for the adhoc implementation in Fourier. It will be generally addressed in https://github.com/phetsims/sun/issues/697. 

Closing.
